CLI/3. Tutorials/2. 部署自動化 AIML pipeline.md

```diff
@@ -1 +1,215 @@
+
+```
+Image：Ubuntu 20.04
+Basic settings：v.super
+```
+
+### Step 2. Enable TWCC CLI
+
+Please refer [1-3. Enter TWCC_CLI environment and start using service](https://man.twcc.ai/@twccdocs/twcc-cli-v05#1-3-%E9%80%B2%E5%85%A5-TWCC_CLI-%E7%92%B0%E5%A2%83%E4%B8%A6%E9%96%8B%E5%A7%8B%E4%BD%BF%E7%94%A8%E6%9C%8D%E5%8B%99) to enable TWCC CLI.
+
+### Step 3. Install jq
+jq is a processing tool in JSON format. TWCC CLI data can be converted to JSON format to show, and some data can be intercepted by jq for application. Related documents can be referred to [jq manual](https://stedolan.github.io/jq/manual/).
+
+-Enter the following command to install jq：
+```bash
+$ sudo apt install jq
+```
+
+
+
+## SSH Without Password
+
+Generally, SSH connection needs to insert the key or enter the password to log in, so the automated process will stop and cannot continue.
+
+Use the following method to create a container and send the SSH key of the Virtual Computing Instance to the container. After that, you can skip the action of inserting the key or entering the password and log in to the container directly.
+
+### Step 1. Create SSH key
+- Enter the following command to create SSH key
+```
+$ ssh-keygen -t rsa -n 4096
+```
+
+Press <kbd>Enter</kbd> key to skip the following three questions
+
+```bash
+Enter file in which to save the key (/home/ubuntu/.ssh/id_rsa):
+Enter passphrase (empty for no passphrase):
+Enter same passphrase again:
+```
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_f96f4eb15d06d0905bc8b51f66f4878e.png)
+
+- Enter the following command to create a Interactive Container
+
+```bash
+$ twccli mk ccs -gpu 1
+```
+
+
+### Step 2. Insert SSH key to Interactive Container
+- Enter the following command to insert SSH key to Interactive Container
+
+```bash
+$ ssh-copy-id <USERNAME>@<CCS_IP> -p <PORT>
+
+#<USERNAME>:supercomputer account
+#<CCS_IP>:Public IP address of the Interactive Container
+#<PORT>:Container SSH's open port
+```
+
+- To insert the key in the container for the first time, you need to enter the **supercomputer password**. After the completion of the Virtual Computing Instance, the container will no longer need to enter the password:
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_4aba0afaefcbda5d00554ed19da64eac.png)
+
+:::info
+<i class="fa fa-paperclip fa-20" aria-hidden="true"></i> **Note:** [<ins>Forgot supercomputer password?</ins>](https://man.twcc.ai/@twccdocs/guide-service-hostname-pwd-otp-zh#%E9%87%8D%E7%BD%AE%E4%B8%BB%E6%A9%9F%E5%AF%86%E7%A2%BC)
+:::
+
+
+### Step 3. Download the sample program:GPU Burn 
+
+This article uses the GPU Burn program as an operating example. You can refer to the following steps to load the required model training program after entering the container environment, or directly [Using SFTP + Filezilla ](https://www.twcc.ai/doc?page=hfs#%E4%BD%BF%E7%94%A8-SFTP--Filezilla-%E5%82%B3%E8%BC%B8%E6%AA%94%E6%A1%88) to upload the program to the container default storage space - Hyper File System.
```

Comment:
[SUG]

This article uses the GPU Burn program as an operating example. > In this article, we use GPU burn testing as an example.